// containers/payments/index.js
import * as R from 'ramda'

import { Button, Card } from 'react-bootstrap'
import React, {Component} from 'react'
import {
  getBasketscannersWithCount,
  getTotalBasketPrice
} from 'selectors'

import BasketCart from 'components/basketCart'
import { Link } from 'react-router-dom'
import MasterCard from './pays/MasterCard.svg' // relative path to image 
import Qiwi from './pays/Qiwi.png' // relative path to image 
import Visa from './pays/Visa.png' // relative path to image 
import Webmoney from './pays/webmoney.png' // relative path to image 
import manual from './pays/manual.jpg'
import scanner from '../scanner/index'

function delete_forms(){
  let forms = document.querySelectorAll(".input__item")
  forms.value = "";
  alert('спасибо за покупку!')
}
export default function Payments(props) {
    return (
      
      <div className="container">
        <div className="top">
          <div className="top__text"> Введите данные оплаты</div>
          <Link to="/basket" ><button className="top__btn ">← Корзина </button></Link>
        </div>
        <div className="cont">
        <div className="form">
          
          <div className="top__name">
            <input className="top__name-name input__item" placeholder="Имя"></input>
            <input className="top__name-sur input__item" placeholder="Фамилия"></input>
          </div>
          <div className="phone">
          <input className="input__item" type="tel" placeholder="телефон"></input>
          </div>
          <div className="paym">
            <select className="paym__select">
              <option>Выберите платежную систему</option>
              <option>MasterCard</option>
              <option>Qiwi</option>
              <option>Webmoney</option>
              <option>Visa</option>
            </select>
          </div>
          <div className="card__number">
            <input className="card__number-one input__item" placeholder="Номер карты" onkeypress="return checkDigit(event)"></input>
            <input className="card__number-two input__item" placeholder="Срок Д."></input>
            <input className="card__number-three input__item" placeholder="CVV"></input>
          </div>
          <button onClick={delete_forms}>Оплатить</button>
        </div>
        <img className="pay_img" src={manual}/>
        </div>
        
      </div>
      
      
    )
}
function check(){
  var form = document.querySelectorAll(".input__item")
  var arr = [...form].forEach((el)=>{
    if(el.value == ""){
      el.classList.add(".red")
    }
  })
}
function checkDigit(event) {
  var allowedChars = "0123456789";
    var entryVal = document.getElementById('txt_cardNumber').value();
    var flag;

    for(var i=0; i<entryVal.length; i++){       
        flag = false;

        for(var j=0; j<allowedChars.length; j++){
            if(entryVal.charAt(i) == allowedChars.charAt(j)) {
                flag = true; 
            }
        }

        if(flag == false) { 
            entryVal = entryVal.replace(entryVal.charAt(i),""); i--; 
        }
    }

    return true;
}
const mapStateToProps = state => {
  return {
    scanners: getBasketscannersWithCount(state),
    totalPrice: getTotalBasketPrice(state)
  }
}
