// components/search/index.js
import React, {Component} from 'react'

import $ from 'jquery';
import {connect} from 'react-redux'
import {searchScanner} from 'actions'

function del(){
  let form = document.querySelector(".form-control");
  let kek = document.querySelector(".form-control-new");
  let butt = document.querySelector(".btn-sum");
  form.value = "";
  form.focus();
  butt.click();
}
class Search extends Component {
  
  constructor(props) {
    super(props)

    this.state = {
      value: ''
    }

    this.handleChange = this.handleChange.bind(this)
    this.handleSubmit = this.handleSubmit.bind(this)
  }

  handleChange(event) {
    this.setState({
      value: event.target.value
    })
  }

  handleSubmit(event) {
    event.preventDefault()
    this.props.searchScanner(this.state.value)
  }

  render() {

    return (
      
      <div className='well blosd'>
        <h3 className='lead'>Si-Shop</h3>
        <div className='input-group'>
          <form onSubmit={this.handleSubmit}>
            <input
              onChange={this.handleChange}
              type='text'
              className='form-control form-control-new'
            />
          </form>
          <span className='input-group-btn'>
          
            <button className='btn btn-default btn-sum' onClick={this.handleSubmit}>
              <span className='glyphicon glyphicon-search'></span>
            </button>
          </span>
        </div>
      </div>
    )
  }
}

const mapDispatchToProps = {
  searchScanner
}

export default connect(null, mapDispatchToProps)(Search)
