// api/mockScanners.js
export default [
    {
        id: "1",
        categoryId: "1",
        name: "GREENVALLEY LIAIR V", Модель: "GREENVALLEY LIAIR V", 
        description: "Воздушный лазерный сканер GreenValley LiAir V – это система лазерного сканирования на основе лидара Livox Mid-40 общим весом 1,3/2,8 кг. LiAir V может быть установлена на DJI M600, M200 и другие платформы, имеет небольшой размер и является самым компактным решением в линейке GVI LiAir. Широко используется для получения трехмерной пространственной информации в таких областях, как геодезия и картография, электроэнергетика, лесное хозяйство, сельское хозяйство, землеустройство, геологические катастрофы и безопасность. LiAir V дополняется программным обеспечением LiPlan, LiAcquire. LiPlan – дополнительное программное обеспечение для системы Lidar LiAir, которое может осуществлять взлет и посадку одной кнопкой, автоматическое выполнение маневра калибровки по фигуре 8, в реальном времени автоматически собирать данные и изображения в соответствии с запланированным путем, а также поддерживать параметры состояния LiAir. Интерфейс LiPlan прост и удобен, и для начинающих легко ориентироваться и выполнять задачи полета и сбора данных одним человеком. LiAcquire - это программное обеспечение для сбора данных, управления устройством и отображения данных в режиме реального времени. Интегрирует модуль постобработки GNSS / INS LiNav, который предоставляет инструменты для обработки данных GNSS / INS, полученных системой LiAir и может генерировать результаты позиционирования с сантиметровой точностью. Есть возможность выводить отчет о точности. Пользователи могут использовать программное обеспечение LiAcquire для выполнения автономной обработки данных или полной онлайн-обработки данных на основе платформы облачного сервиса LiCloud. LiCloud - универсальная облачная платформа, разработанная GreenValley International для управления, обработки и визуализации данных, а также технической поддержки GVI.",
        price: 1600000, Цена: 1600000,
        image: "/uploads/1.jpg",
        range: "260 метров", Дальность: "260 метров",
        speed: "100 000 т/сек", Скорость: "100 000 т/сек",
        weith: "1.3 кг", Вес: "1.3 кг",
        angle: "38.4°", Обзор: "38.4°",
    },
    {
        id: "2",
        categoryId: "2",
        name: "LEICA BLK360", Модель: "LEICA BLK360",
        description: "Лазерный сканер Leica BLK360 – это компактный прибор, предназначенный для получения облаков точек окружающей местности с точностью до 6 мм на 10 м. Управление осуществляется как с помощью многофункциональной кнопки на корпусе прибора, так и с помощью планшета с установленным программным обеспечением Recap Pro, которое легко освоить даже без предварительной подготовки.Прибор управляется как с помощью многофункциональной кнопки, так и через подключенный планшет с установленным на нем программным обеспечением Autodesk ReCap Pro Mobile. Графический интерфейс и понятная структура меню помогут грамотно провести съемку объекта наземным лазерным сканером Leica BLK360 при любом уровне подготовки специалиста. Программа автоматически рассчитывает нужные параметры и конвертирует сшитое облака точек в удобный формат rcp.",
        price: 1800000, Цена: 1800000,
        image: "/uploads/2.jpg",
        range: "10 метров", Дальность: "10 метров",
        speed: "до 360 000 т/сек", Скорость: "до 360 000 т/сек",
        weith: "1 кг", Вес: "1 кг",
        angle: "40°", Обзор: "40°",
    },
    {
        id: "3",
        categoryId: "1",
        name: "LEICA ALS80", Модель: "LEICA ALS80",
        description: "Воздушный лазерный сканер Leica ALS80 – это универсальное модульное решение, с помощью которого выполняется сканирование с борта летательного аппарата на различной высоте. Устройство может быть адаптировано для сбора плотного облака точек при пролете на небольшой высоте или сканирования больших пространств в масштабах региона или государства на высоте до 5 км.За управление полетом отвечает система Leica FlightPro, которая получает данные о полете с ПО Leica MissionPro. В этом ПО осуществляется планирование полетов. Пользователь может вручную или автоматически оптимизировать настройки.",
        price:4600000, Цена:4600000,
        image: "/uploads/3.jpg",
        range: "от 100 метров", Дальность: "от 100 метров",
        speed: "до 100 000 т/сек", Скорость: "до 100 000 т/сек",
        weith: "47 кг", Вес: "47 кг",
        angle: "30°", Обзор: "30°",
    },
    {
        id: "4",
        categoryId: "2",
        name: "LEICA RTC360", Модель: "LEICA RTC360",
        description: "Лазерный сканер Leica RTC360 – это компактный по размеру и высокопроизводительный прибор для трехмерного сканирования местности и объектов. Он выполняет измерения со скоростью 2 млн точек в секунду и имеет дальность работы до 130 метров при 90% отражающей способности поверхностей. Поддерживается технология оцифровки сигнала WFD, реализованы алгоритмы автоматической сшивки результатов сканирования.Сканер может использоваться в геодезических изысканиях, в промышленности и для сбора данных об инфраструктурных объектах различного назначения. Трехмерное сканирование позволяет быстро получить максимальное количество данных в минимальные сроки и оперативно отслеживать выполненные работы на стройплощадке. Также, полученные данные можно использовать для создания BIM-моделей. Особенность этих моделей заключается в том, что они содержат трехмерные чертежи всех частей зданий и инженерных систем. С их помощью удобно мониторить строительство, реконструкцию и техническое обслуживание зданий.",
        price: 5350000, Цена: 5350000,
        image: "/uploads/4.jpg",
        range: "от 130 метров", Дальность: "от 130 метров",
        speed: "до 300 000 т/сек", Скорость: "до 300 000 т/сек",
        weith: "5 кг", Вес: "5 кг",
        angle: "56°", Обзор: "56°",
    },
    {
        id: "5",
        categoryId: "1",
        name: "MOCCA KIT-330", Модель: "MOCCA KIT-330",
        description: "Наземный лазерный сканер KIT - является комбинацией роботизированного тахеометра и лазерного трехмерного сканера. Инструмент обладает односекундной точностью угловых измерений, а скорость сканирования достигает до 26 600 в секунду при диапазоне 600 м. Trimble SX10 выполняет сканирование 3D данных с высокой плотностью и скоростью до 26,600 точек в секунду, обеспечивая высокую точность во всем диапазоне измерений - до 600 м. Благодаря новой технологии Trimble Lightning 3DM, для высокоточных тахеометрических измерении и высокоскоростного 3D сканирования используется всего один инструмент, обладающий невиданной ранее производительностью. Инструмент создан с использованием надежных и мощных технологий, таких как MagDrive™ и SurePoint™. Объединение с новейшими технологиями, такими как Autolock, позволяет SX10 выполнять подробную съемку с максимальной точностью. А благодаря полной интеграции SX10 с программным обеспечением Trimble Access™ и Trimble Business Center, ваши полевые бригады смогут быстро выполнять работы, используя хорошо знакомые и эффективные технологические процессы.",
        price: 3400000, Цена: 3400000,
        image: "/uploads/5.jpg",
        range: "от 100 метров", Дальность: "от 100 метров",
        speed: "до 150 000 т/сек", Скорость: "до 150 000 т/сек",
        weith: "4 кг", Вес: "4 кг",
        angle: "68°", Обзор: "68°",
    },
    {
        id: "6",
        categoryId: "1",
        name: "MOCCA KUAC-250", Модель: "MOCCA KUAC-250",
        description: "Мобильный лазерный сканер оснащен камерами, которые обеспечивают получение панорамных снимков общим разрешением в 30 Мп. Фотографирование возможно во время движения автомобиля благодаря высокой скорости записи. Это дает возможность собрать необходимую информацию на полевом этапе исследований в кратчайшие сроки и сократить стоимость исследований. Установлено 6 камер, которые дают обзор на 360°.Система оснащена модулем точного позиционирования на базе GNSS. Кроме того, внутри лазерного сканера Trimble MX7 установлены инерциальные модули Applanix IMU-69, которые повышают точность привязки собранной информации. Наличие IMU-датчиков позволяет сохранять данные о положении при отсутствии навигационных данных со спутника.",
        price: 3400000, Цена: 3400000,
        image: "/uploads/6.jpg",
        range: "от 100 метров", Дальность: "от 100 метров",
        speed: "до 150 000 т/сек", Скорость: "до 150 000 т/сек",
        weith: "4 кг", Вес: "4 кг",
        angle: "68°", Обзор: "68°",
    },
    {
        id: "7",
        categoryId: "1",
        name: "LEICA ALS80", Модель: "LEICA ALS80",
        description: "Система мобильного картографирования LEICA ALS80 - это законченное решение для мобильного лазерного сканирования, сочетающее в себе уникальное современное оборудование, интуитивно понятное полевое ПО и надежный программный пакет для постобработки данных. Система Trimble MX9 легко устанавливается на транспортное средство. Она быстро получает плотные облака точек и панорамные снимки. В системе установлены два высокоскоростных сканера с дальностью сканирования до 420 м. Решение оснащено 4-мя камерами: панорамной, двумя боковыми и одной, направленной назад и вниз для детальной съемки дорожного полотна. Навигация и позиционирование Trimble MX9 осуществляется при помощи системы Applanix AP60 или AP40. Быстрая и высококачественная коридорная съемка может проводиться на оживленных магистралях со скоростью до 110 км/ч.",
        price: 4600000, Цена: 4600000,
        image: "/uploads/7.jpg",
        range: "от 100 метров", Дальность: "от 100 метров",
        speed: "до 100 000 т/сек", Скорость: "до 100 000 т/сек",
        weith: "47 кг", Вес: "47 кг",
        angle: "30°", Обзор: "30°",
    },
    // {
    //     id: "8",
    //     categoryId: "2",
    //     model: "LEICA RTC360",
    //     description: "Лазерный сканер Leica RTC360 – это компактный по размеру и высокопроизводительный прибор для трехмерного сканирования местности и объектов",
    //     price: 5350000,
    //     image: "/uploads/8.jpg",
    //     range: "от 130 метров",
    //     speed: "до 300 000 т/сек",
    //     weith: "5 кг",
    //     angle: "56°",
    // },
    // {
    //     id: "9",
    //     categoryId: "1",
    //     model: "LEICA RTC360",
    //     description: "Лазерный сканер Leica RTC360 – это компактный по размеру и высокопроизводительный прибор для трехмерного сканирования местности и объектов",
    //     price: 5350000,
    //     image: "/uploads/9.jpg",
    //     range: "от 130 метров",
    //     speed: "до 300 000 т/сек",
    //     weith: "5 кг",
    //     angle: "56°",
    // },
    // {
    //     id: "10",
    //     categoryId: "1",
    //     model: "MOCCA KIT-330",
    //     description: "Лазерный сканер MOCCA KIT-330 – компактный и незаменимый прибор в движении",
    //     price: 3400000,
    //     image: "/uploads/10.jpg",
    //     range: "от 100 метров",
    //     speed: "до 150 000 т/сек",
    //     weith: "4 кг",
    //     angle: "68°",
    // },
    // {
    //     id: "11",
    //     categoryId: "1",
    //     model: "MOCCA KUAC-250",
    //     description: "Лазерный сканер MOCCA KIT-330 – компактный и незаменимый прибор в движении",
    //     price: 3400000,
    //     image: "/uploads/11.jpg",
    //     range: "от 100 метров",
    //     speed: "до 150 000 т/сек",
    //     weith: "4 кг",
    //     angle: "68°",
    // },
    // {
    //     id: "12",
    //     categoryId: "2",
    //     model: "GREENVALLEY LIAIR V",
    //     description: "Воздушный лазерный сканер GreenValley LiAir V – это система лазерного сканирования на основе лидара Livox Mid-40 общим весом 1,3/2,8 кг",
    //     price: 1600000,
    //     image: "/uploads/12.jpg",
    //     range: "260 метров",
    //     speed: "100 000 т/сек",
    //     weith: "1.3 кг",
    //     angle: "38.4°",
    // },
]
