// containers/scanner/index.js
import * as R from 'ramda'

import React, {Component} from 'react'
import {addScannerToBasket, fetchScannerById} from 'actions'

import BasketCart from 'components/basketCart'
import {Link} from 'react-router-dom'
import {connect} from 'react-redux'
import {getScannerById} from 'selectors'

class Scanner extends Component {
  componentDidMount() {
    const scannerId = this.props.match.params.id
    this.props.fetchScannerById(scannerId)
  }

  renderField() {
    const {scanner} = this.props
    const columnField = R.compose(
      R.toPairs,
      R.pick([
        'Модель',
        'Цена',
        'Дальность',
        'Скорость',
        'Вес',
        'Обзор'
      ]),
    )(scanner)

    console.log('columns', columnField)

    return columnField.map(([key, value]) => (
      <div className='column' key={key}>
        <div className='ab-details-title'>
          <p>{key}</p>
        </div>
        <div className='ab-details-info'>
          {value}
        </div>
      </div>
    ))
  }

  renderContent() {
    const {scanner} = this.props

    return (
      <div className='thumbnail '>
        <div className='row'>
          <div className='col-md-6'>
            <img
              className='img-thumbnail'
              src={scanner.image}
              alt={scanner.name}
            />
          </div>
          <div className='col-md-6'>
            {this.renderField()}
          </div>
        </div>
        <div className='caption-full'>
          
          <h4>{scanner.name}</h4>
          <p>{scanner.description}</p>
        </div>
      </div>
    )
  }

  renderSidebar() {
    const {scanner, addScannerToBasket} = this.props

    return (
      <div>
        <p className='lead'>Si-Shop</p>
        <BasketCart/>
        <div className='form-group'>
          <h1>{scanner.name}</h1>
          <h2>{scanner.price} ₽</h2>
        </div>
        <Link to='/' className='btn btn-info btn-block'>
          Вернуться к покупкам
        </Link>
        <button
          type='button'
          className='btn btn-success btn-block'
          onClick={() => addScannerToBasket(scanner.id)}
        >
          Добавить в корзину
        </button>
      </div>
    )
  }

  render() {
    return (
      <div className='view-container'>
        <div className='container'>
          <div className='row'>
            <div className='col-md-9'>
              {this.props.scanner && this.renderContent()}
            </div>
            <div className='col-md-3'>
              {this.props.scanner && this.renderSidebar()}
            </div>
          </div>
        </div>
      </div>
    )
  }
}

const mapStateToProps = state => {
  return {
    scanner: getScannerById(state, state.scannerPage.id)
  }
}

const mapDispatchToProps = {
  fetchScannerById,
  addScannerToBasket
}

export default connect(mapStateToProps, mapDispatchToProps)(Scanner)
